src/tools/tidy/src/ui_tests.rs

```diff
@@ -11,7 +11,7 @@ use std::path::{Path, PathBuf};
 const ENTRY_LIMIT: usize = 900;
 // FIXME: The following limits should be reduced eventually.
 const ISSUES_ENTRY_LIMIT: usize = 1854;
-const ROOT_ENTRY_LIMIT: usize = 865;
+const ROOT_ENTRY_LIMIT: usize = 866;
```

Comment:
Bumping this by 1 because I'm adding a new directory for a feature (`-Cinstrument-coverage`) that I believe currently has no standalone UI tests.